### DISCUSSION TITLE
Kullanıcıya kürenin çapı sorulur, gelen yanıta göre hacmi hesaplanır.

### DISCUSSION THREAD

**Discussion by DolphyWind** [sonsuzus/sonsuzus.github.io#4] [Soru Cevap] [answered]
Kullanıcıya kürenin çapı sorulur, gelen yanıta göre hacmi hesaplanır. C Programlama, python programlama vs. ile cevapları.

**Comment by DolphyWind** [comment, marked as answer]
```
#include <stdio.h>

int main()
{
	float r, pi=3.1415926535;
	printf("Kurenin yaricapini girin: ");
	scanf("%f", &r);
	
	float hacim = (4.0f/3.0f)*pi*r*r*r;
	printf("Kurenin hacmi: %f", hacim);
	return 0;
}
```
C Programlama

**Comment by BerraAlkann** [comment]
r = input("Kurenin Capini Giriniz: ")
r = int(r)
pi = 3.1415926535
print("Kurenin hacmi:", ((4 * pi * ((r/2)*(r/2)*(r/2))/3)))

**Reply by sonsuzus** [reply]
Code satırları kullanmamışsın.

**Comment by CinarSatir** [comment]
```
#Çınar Satır
print(" KüreHacim programına hoş geldiniz\nBu programda kürenizin hacmini bulabilirsiniz.")
R = float(input("Kürenizin capini giriniz : "))
c = input("Kürenizin capının cinsi nedir(metre icin m santimetre icin cm) : ")
r = R/2
pi = 3.14159
h = 4/3*pi*(r*r*r)
if (c == "m") or (c == "cm"):
    print("Kürenizin hacmi :",h,c,"küptür.")
else :
    print("Yanlış kodlama yaptınız")
```

**Reply by sonsuzus** [reply]
Kod hatalı
Ayrıca Code satırları kullanmamışsın.

**Comment by AhmetEmirErsoz** [comment]
r = input("Kürenin çapını yaz: ")
r = int(r)
pi = 3.14
print("Kürenizin Hacmi şuna eşittir:", ((4 * pi * ((r/2)(r/2)(r/2))/3)))

**Reply by sonsuzus** [reply]
Code satırları kullanmamışsın.

**Comment by AoiKos1243** [comment]
print("Merhaba Ben Kürenin çapından hacmini hesaplayabilirim") 
r = float(input("İnanmıyormusun o zaman bana bir çap yaz:"))
r = int(r)
p = 3.14
print("HAHA BULDUM işte cevap şu:",((4 * pi * ((r/2)(r/2)(r/2))/3)))

**Reply by sonsuzus** [reply]
Code satırları kullanmamışsın.

**Comment by onursonkaya** [comment]
r= float(input("kürenin çapını giriniz"))
pi=3.14
print("kürenin hacmi",(4/3)*pi*r*r*r)

**Reply by sonsuzus** [reply]
Kod hatalı, ayrıca Code satırları kullanmamışsın.

**Comment by Sadik1Dev** [comment]
```
r = input("lütfen kürenin yarı çapını girin: ")
r = int(r)
pi = 3.1415926535
r3 = r*r*r
hacim = 4/3*pi*r3
print("kürenizin hacmi: ",hacim)
```

**Comment by akgembi12** [comment]
```
pi=3.1415
r=int(input("kürenin yarıçapını girin (birim cinsinden)"))
alan=(4*pi)*(r*r)
hacim=(4/3)*pi*(r*r*r)
print("kürenin hacmi",hacim,"birimküp, alanı ise",alan,"dır")
```

**Comment by OmerAkbayrak** [comment]
```python

R = input("Kürenin Çapını Giriniz:")
R = float(R)
r=R/2
print("Kürenin Hacmi ",r*r*r ,"dir")
```

**Reply by sonsuzus** [reply]
Formül yanlış

**Comment by TahaGuzell** [comment]
R=input("Cemberin capini giriniz: ")
R=int(R)
r=R/2
pi=3.14
cevre=2*pi*r
alan=pi*r*r
print("Cemberin alani",alan,". Cemberin cevresi",cevre)

**Reply by sonsuzus** [reply]
Hatalı kod.

**Comment by KayraMertAydin** [comment]
R = input ("Lütfen Kürenin Çapını Giriniz : ")
R = float(R)

r = R/2
r = float (r)

print ("Çapı", R, "Olan Bir Kürenin Hacmi ", (r*r*r)*3.1415*(4/3))

**Reply by sonsuzus** [reply]
Hatalı kod.

**Comment by Musabkaracinar** [comment]
``yc = input("yaricap giriniz: ")
yc = int(yc)
pi = 3.1415926535
yc3 = yc*yc*yc
h = 4/3*pi*yc3
print("kurenizin hacmi: ",h)

**Reply by sonsuzus** [reply]
Hatalı kod.

**Comment by Ersan0102** [comment]
'cap = input("Çap ne?: ")
cap = float(cap)
yaricap = float(cap/2)
pi = float(3.14)
Hacim = yaricap*yaricap*yaricap*4/3*pi
print("Bunun hacmi",Hacim)'

**Reply by sonsuzus** [reply]
Hatalı kod.

**Comment by Ameth28** [comment]
cap = input("Kürenin çapını giriniz: ")
cap = int(cap)
yaricap = cap/2
r3 = yaricap * yaricap * yaricap
pisayisi = 3.14
print(cap, "çaplı kürenin hacmi:", (4/3) * pisayisi * r3)

- Ahmet Emre Dağ

**Reply by sonsuzus** [reply]
Code satırına almalısın kodları.

**Comment by AdaNilPoyraz** [comment]
cap = input("Kürenin Çapını Giriniz:")
cap = float(cap)
pi = 3.14
hcm = (4/3) * pi * ((cap/2) * (cap/2) * (cap/2))
print("==Kürenin Hacmi==")
print(hcm)

>Ada Nil Poyraz

**Reply by sonsuzus** [reply]
Code satırına almalısın kodları.

**Comment by CinarDumanli12** [comment]
```
r = input("Kürenin yarı çapını girin-> ")
r = int(r)
Pi = 3.14
r3tane = r*r*r
hacim = 4/3*Pi*r3tane
print("Küre hacmi-> ",hacim)

```

**Comment by OmerB2** [comment]
``` 
R = input("Kürenin çapını giriniz:")
R = int (R)
r = R/2
rküp = r*r*r
pi = 3.14159265359
print ("Kürenin hacmi=",4/3*pi*rküp)
```

**Comment by Barlasgng** [comment]
r = input("yarı çap girin")
r = int(r)
Pi = 3.14
r3 = r*r*r
hacim = 4/3*pi*r3
print("kürenizin hacmi: ",hacim")

**Comment by adasue** [comment]
`cap = input("Kurenin capini giriniz: ")
cap = float(cap)
print("Kurenin hacmi: ", 4/3*3.14*cap)`

**Comment by YagizEfeZ** [comment]
r = input("kürenin çapını gir:")
r = int(r)
pi = 3.14
print("kürenin hacmi:",((4 * pi * ((r/2)(r/2)(r/2))/3)))

**Comment by Alperg123456** [comment]
cap = input("kürenin çapını giriniz")
cap = int(cap)
r = (cap/2)
pi = 3.14
r3 = r*r*r
hacim = (4/3*pi*r3)
print("girdiğiniz",cap,"çaplı kürenin hacmi = ",hacim)

**Comment by ElifNurSoyler** [comment]
R=int(input("Kürenin Çapını Giriniz: "))
r=R/2
pi=3.14
r3=r*r*r
hacim=4/3*pi*r3
print("Kürenin Hacmi: ",hacim)

**Comment by dudusoo** [comment]
a=input (float("Çapı giriniz: "))
b=float (a/2)
c=float ((4/3)*3.14*b*b*b)
print ("Hacim: ",c)

**Comment by Berkek007** [comment]
cap = float (input("Kürenin çapını giriniz: "))
print("Bu kürenin hacmi",4/3*3.14*cap/2*cap/2*cap/2)

**Comment by Adaisland** [comment]
`yarıçap=input("Kürenizin yarı çapını giriniz:")
yarıçap=int(yarıçap)
Pi=3.14
Çarpım=yarıçap*yarıçap*yarıçap
Hacim=4/3*Pi*Çarpım
print("Kürenizin hacmi:",Hacim)`

**Comment by AcelyaP** [comment]
```
r = input ("Kürenin yarı çapını nedir?")
r = int(r)
Pi = 3.14
r3tane = r*r*r
print ("Kürenin hacmi:", 4/3*Pi*r3tane)
```
